### PR TITLE
zh-cn translate abortsignal/timeout

### PR DESCRIPTION
**Pull request by fwqaaq**

### CODE REVIEW

File: files/zh-cn/web/api/abortsignal/timeout/index.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+---
+title: AbortSignal.timeout()
+slug: Web/API/AbortSignal/timeout
+---
+{{APIRef("DOM")}}
+
+静态方阿飞 **`AbortSignal.timeout()`** 返回一个指定时间后将自动中止的 {{domxref("AbortSignal")}}。
```
Comment (yin1999) [inline files/zh-cn/web/api/abortsignal/timeout/index.md:7]: 请修改

File: files/zh-cn/web/api/abortsignal/timeout/index.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+信号在超时时使用 `TimeoutError` {{domxref("DOMException")}} 中止，或者由于按下一个浏览器停止按钮（或者一些内置的“停止”操作）而使用 `AbortError` {{domxref("DOMException")}} 中止。这允许 UI 区分超时错误（通常需要通知用户）和用户触发的错误（不需要通知用户）。
+
+超时将基于活动的时间，而不是经过的时间，如果代码在挂起的工作中运行或者文档在向后缓存中时（“[bfcache](https://web.dev/bfcache/)”），将有效的暂停。
+
+> **备注：** 在撰写代码时，无法组合多个。意思是你不能使用超时的 signal 或者通过调用 {{domxref("AbortController.abort()")}} 直接中止下载。
```
Comment (yin1999) [inline files/zh-cn/web/api/abortsignal/timeout/index.md:13]: 撰写文章-编写代码：

```suggestion
> **备注：** 在编写代码时，无法组合多个信号。意思是你不能使用超时的 signal 或者通过调用 {{domxref("AbortController.abort()")}} 直接中止下载。
```

**Review by yin1999 — CHANGES_REQUESTED** [review]

File: files/zh-cn/web/api/abortsignal/timeout/index.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+title: AbortSignal.timeout()
+slug: Web/API/AbortSignal/timeout
+---
+{{APIRef("DOM")}}
+
+静态方法 **`AbortSignal.timeout()`** 返回一个指定时间后将自动中止的 {{domxref("AbortSignal")}}。
+
+信号在超时时使用 `TimeoutError` {{domxref("DOMException")}} 中止，或者由于按下一个浏览器停止按钮（或者一些内置的“停止”操作）而使用 `AbortError` {{domxref("DOMException")}} 中止。这允许 UI 区分超时错误（通常需要通知用户）和用户触发的错误（不需要通知用户）。
+
+超时将基于活动的时间，而不是经过的时间，如果代码在挂起的工作中运行或者文档在向后缓存中时（“[bfcache](https://web.dev/bfcache/)”），将有效的暂停。
```
Comment (yin1999) [inline files/zh-cn/web/api/abortsignal/timeout/index.md:11]: worker is not `工作`，https://developer.mozilla.org/zh-CN/docs/Web/API/Web_Workers_API/Using_web_workers

Would you mind to check the english words after translation?

**Review by yin1999 — CHANGES_REQUESTED** [review]

File: files/zh-cn/web/api/abortsignal/timeout/index.md (hunk 3 above)
Comment (Jack-Zhang-1314) [reply files/zh-cn/web/api/abortsignal/timeout/index.md:11]: yeah, you are right

Comment (Jack-Zhang-1314) [reply files/zh-cn/web/api/abortsignal/timeout/index.md:11]: Maybe it should not be translated

Comment (Jack-Zhang-1314) [inline files/zh-cn/web/api/abortsignal/timeout/index.md:11]:
```suggestion
超时将基于活动的时间，而不是经过的时间，如果代码在指定的的工作线程中运行或者文档在向后缓存中时（“[bfcache](https://web.dev/bfcache/)”），将有效的暂停。
```

Comment (yin1999) [reply files/zh-cn/web/api/abortsignal/timeout/index.md:11]: do not translate `worker`

Comment (Jack-Zhang-1314) [reply files/zh-cn/web/api/abortsignal/timeout/index.md]: 👌

**Review by yin1999 — APPROVED** [review]

**Pull request merged**